src/gles_fns.rs

```diff
@@ -681,6 +681,45 @@ impl Gl for GlesFns {
         panic!("not supported");
     }
 
+    fn copy_image_sub_data(&self,
+                           src_name: GLuint,
+                           src_target: GLenum,
+                           src_level: GLint,
+                           src_x: GLint,
+                           src_y: GLint,
+                           src_z: GLint,
+                           dst_name: GLuint,
+                           dst_target: GLenum,
+                           dst_level: GLint,
+                           dst_x: GLint,
+                           dst_y: GLint,
+                           dst_z: GLint,
+                           src_width: GLsizei,
+                           src_height: GLsizei,
+                           src_depth: GLsizei) {
+        if self.ffi_gl_.CopyImageSubDataEXT.is_loaded() {
```

Comment:
Why guard? Shouldn't it just panic if the function isn't loaded?

Reply:
Panicing sounds like the best option. Is that what happens by default if I drop the guard? Or do I have to manually check if it's not loaded and panic?